### ISSUE TITLE
Chrome: Default scrollbar on some screen-sizes is not hidden correctly

### ISSUE THREAD

**Issue by Danilqa** [malte-wessel/react-custom-scrollbars#225]
![screen](https://user-images.githubusercontent.com/2026837/35181145-9b4517b2-fdcd-11e7-8806-a12a1314bb8e.JPG)

If I customize default scrollbar by "::-webkit-scrollbar", I'll get this:
![image](https://user-images.githubusercontent.com/2026837/35181166-e31d14fe-fdcd-11e7-814f-a599dcf32a54.png)

Unforeseen shifting to right.

**Comment by RoboQuasar** [comment]
Have the same problem
![2018-02-09 11 53 49](https://user-images.githubusercontent.com/20340783/36019346-0684f3b8-0d90-11e8-83e6-64cbd02ba01a.png)

I can resolve it by using attribute like this :
 `renderView={({ style, ...props }) =>
                <div {...props} style={{ ...style, marginBottom: '-16px' }} />} `

, but i suppose, that a not universal decision

**Comment by OscarCanek** [comment]
I have the same problem with Chrome

![image](https://user-images.githubusercontent.com/3521524/36103064-e0c560a4-0fd3-11e8-9104-02546a3af8a8.png)

![image](https://user-images.githubusercontent.com/3521524/36103076-edf4f3d4-0fd3-11e8-9af8-3f19e40ee21a.png)

How I can fix that?

**Comment by OscarCanek** [comment]
Any update on this?

**Comment by ev01ve** [comment]
I'm seeing this problem in Chrome too.

**Comment by tungvn** [comment]
I see this problem too.
But I can resolve this issue by using:
```
renderView={({ style, ...props }) => <div {...props} style={{ ...style, marginRight: '-18px' }} />}
```
I think in some specific cases, the width or height of scrollbar could not be calculated truthly and it made our problems.

**Comment by mustofa-binary** [comment]
have the same problem in chrome, @tungvn renderview is a props inside scrollbar?

**Comment by tungvn** [comment]
> have the same problem in chrome, @tungvn renderview is a props inside scrollbar?

Yes. See the documentation at https://github.com/malte-wessel/react-custom-scrollbars#usage

**Comment by mustofa-binary** [comment]
@tungvn thank you, it works for me on windows chrome, however, on mac, there is no scrollbar shown at all. i think PR is needed to fix this

**Comment by tungvn** [comment]
@mustofa-binary I am really not sure the problem is, so I think just treat like that first

**Comment by Nichon4** [comment]
I have met same problem.
as i see, this happen when in os setted option "Make everything bigger".
default scroll width(17px) has problems with round after zoom. 
when i set global scrollbar width to 20px it fix the problem.

upd: function getScrollbarWidth() calculate scrollbar width by creating div in document root. if you reassign scrollbar width somewhere in your app, react-custom-scrollbars won't know about it.
